### ISSUE TITLE
Getting values from page#property 

### ISSUE THREAD

**Issue by DavideMontersino** [amir20/phantomjs-node#405]
It's impossible to get values out of page#property -or I am getting something wrong.

### Actual behavior

``` javascript

myObj = {};

page.property('onResourceRequested', function(req, networkRequest) {
    // console.log('onResourceRequested', req, networkRequest);
    myObj.req = req;
    myObj.networkRequest = networkRequest;
  }, myObj);
```
If after I try to access `myObj`, it will still be an empty object.

Am I wrong, or the only possible operations inside _page#property_ are basically using some console.log ?

With version 1.0 I was able to use phantomjs-node to retrieve requests and save them as HAR into a mongodb. This seems no longer possible..

- OS: Ubuntu
- Node version (`node -v`): 4.4.0
- package version:

**Comment by amir20** [comment]
See #377. I am working on a better solution to make this easier.

**Comment by amir20** [comment]
I actually like how you suggested using an `{}` to get data out. I might look at that for a better solution. :)

**Comment by amir20** [comment]
I am proposing this as a fix #406. Can you provide feedback @DavideMontersino ? I could appreciate your feedback.

**Comment by amir20** [comment]
This is fixed by #406.

**Comment by DavideMontersino** [comment]
Sorry, I was not online this weekend :+1: 

Going to try it out now!
